src/content/docs/en/guides/deploy/cloudflare.mdx

```diff
@@ -137,3 +137,14 @@ If you're encountering errors, double-check the version of `node` you're using l
 Cloudflare requires [Node v16.13](https://miniflare.dev/get-started/cli#installation), which is a more recent version than Astro’s out-of-the-box minimum, so double check you’re using at least v16.13.
 
 Client-side hydration may fail as a result of Cloudflare's Auto Minify setting. If you see `Hydration completed but contains mismatches` in the console, make sure to disable Auto Minify under Cloudflare settings.
+
+If you are deploying or building a project that is using server-side rendering and the server fails to build with error messages similar to:
```

Comment:
I don't think the issue occurs when deploying, it already throws the error during build

Reply:
```suggestion
If you are building a project that is using server-side rendering and the server fails to build with error messages similar to:
```